docs/Translation.md

```diff
@@ -125,6 +135,42 @@ class LocaleSwitcher extends Component {
 export default connect(undefined, { changeLocale: changeLocaleAction })(LocaleSwitcher);
 ```
 
+## Changing the locale using the browser URL 
```

Comment:
Not the best illustration for a custom saga IMO (see other comment on that code snippet)